content/en/blog/_posts/2022-12-12-host-process-containers-stable/index.md

```diff
@@ -0,0 +1,113 @@
+
+Until now, scenarios common to Linux privileged containers, such as kube-proxy (via kubeadm),
+storage, and networking, all required proxy solutions to enable functionality on Windows.
+With HostProcess containers users have direct access to the node without the need for
+any workarounds. Gone are the days where cluster operators would need to manually log onto
+Windows nodes to perform administrative duties - instead replaced with a continuous
+development pipeline.
+
+## How does it work?
+
+Windows HostProcess containers are implemented with Windows _Job Objects_, a break from the
+previous container model using server silos. Job objects are components of the Windows OS which offer the ability to
+manage a group of processes as a group (a.k.a. _jobs_) and assign resource constraints to the
+group as a whole. Job objects are specific to the Windows OS and are not associated with the Kubernetes [Job API](https://kubernetes.io/docs/concepts/workloads/controllers/job/). They have no process or file system isolation,
+enabling the privileged payload to view and edit the host file system with the
+correct permissions, among other host resources. The init process, and any processes
+it launches or that are explicitly launched by the user, are all assigned to the
+job object of that container. When the init process exits or is signaled to exit,
+all the processes in the job will be signaled to exit, the job handle will be
+closed and the storage will be unmounted.
+
+HostProcess and Linux privileged containers enable similar scenarios but differ
+greatly in their implementation (hence the naming difference). HostProcess containers
+have their own pod security policies. Those used to configure Linux privileged
+containers **do not** apply. Enabling privileged access to a Windows host is a
+fundamentally different process than with Linux so the configuration and
+capabilities of each differ significantly. Below is a diagram detailing the
+overall architecture of Windows HostProcess containers:
+
+{{< figure src="hpc-architecture.png" alt="HostProcess Architecture" >}}
+
+Two major features were added prior to moving to stable: the ability to run as local user accounts, and
+a simplified method of accessing volume mounts. For more info visit our [updated documentation page](/docs/tasks/configure-pod-container/create-hostprocess-pod)
+
+## Use Cases
+
+The SIG-Windows community has been busy putting HostProcess containers to use - even before GA! CThey've
+been very excited to use HostProcess containers for a number of important activities that were a pain to
+perform in the past. Now using HostProcess containers you can install the windows-exporter on your nodes
+and collect extremely detailed monitoring information with Prometheus. You can now run csi-proxy, flannel,
+calico, event loggers, and more as HPCs.
+
+To see more HPC examples head over to [HostProcess Examples](https://github.com/kubernetes-sigs/sig-windows-tools/tree/master/hostprocess)
```

Comment:
```suggestion
To see more examples, look at the
[HostProcess examples](https://github.com/kubernetes-sigs/sig-windows-tools/tree/f366740c551b4217887a9eb339fd7ec3d6caacce/hostprocess)
that the Kubernetes project publishes on GitHub.
```

HPC is established jargon for “high performance computing”. Don't write “HPC” to mean “HostProcess Pods”